code/modules/mob/living/simple_animal/friendly/drone/say.dm

```diff
@@ -14,16 +14,29 @@
 /mob/living/simple_animal/drone/get_spans()
 	return ..() | SPAN_ROBOT
 
-/mob/living/simple_animal/drone/proc/alert_drones(msg, dead_can_hear = 0)
+
+
+//Base proc for anything to call
+/proc/_alert_drones(msg, dead_can_hear = 0, list/factions)
```

Comment:
This is an unconventional way to name a proc, why not something like `alert_drones_output` or `alert_drones_message`